Gemfile

```diff
@@ -28,13 +29,17 @@ group :doc do
   # bundle exec rake doc:rails generates the API under doc/api.
   gem 'sdoc', require: false
 end
+group :production do
+  # Use mysql as the database for Active Record
+end
```

Comment:
There's nothing in this group, so I'm confused if the later line changes were meant to go in here? Since those aren't in a group they will still be installed on all installations.